tests/text_processing/tickets_data/PHPBB3-14105.html

```diff
@@ -0,0 +1,2 @@
+<a href="http://localhost/index.php" class="postlink">...</a><br>
+<a href="http://localhost/index.php?foo=1" class="postlink">...</a>
```

Comment:
Which is impossible, since adding a new line in a post results in a newline in the HTML.